src/main/java/frc/robot/commands/CollectCoral.java

```diff
@@ -1,10 +1,16 @@
 package frc.robot.commands;
 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import frc.robot.subsystems.CoralGripper;
+import java.time.Instant;
+
+import java.util.Timer;
 
 public class CollectCoral extends Command {
     public CoralGripper coralGripper;
+    Instant previousTime = null;
```

Comment:
don't use `Instant` as it uses the less trusty system clock. Use `Timer.getFPGATimestamp()` instead.